biztalk/technical-guides/using-the-performance-analysis-of-logs-pal-tool.md

```diff
@@ -44,70 +44,70 @@ The PAL (Performance Analysis of Logs) tool reads in a performance monitor count
+|                 Microsoft BizTalk Server:  BizTalk Host Queue Length                  |                                                                                                                                                                                                                                                                                            The BizTalk Host Queue Length tracks the total number of messages in the particular host queue. You can use length size (i.e., ‘BizTalk:MessageBox:HostCounters:Host Queue – Length’) to give a more detailed view of the number of messages being queued up internally by showing the queue depth for an individual host. This counter can be useful in determining if a specific host is bottlenecked. Assuming unique hosts are used for each transport, this can be helpful in determining potential transport bottlenecks. This analysis checks for average queue lengths greater than 1. The Host Queue Length is a weighted Queue length by aggregating the record count of all the Queues (Work Q, State Q, Suspended Q) of the target host.<br /><br /> Reference: BizTalk Server 2006: Managing a Successful Performance Lab **Note:**  This white paper is also applicable to BizTalk Server.                                                                                                                                                                                                                                                                                             |
```

Comment:
```suggestion
|                 Microsoft BizTalk Server:  BizTalk Host Queue Length                  |                                                                                                                                                                                                                                                                                            The BizTalk Host Queue Length tracks the total number of messages in the particular host queue. You can use length size, for example, **BizTalk:MessageBox:HostCounters:Host Queue – Length**, to give a more detailed view of the number of messages being queued up internally by showing the queue depth for an individual host. This counter can be useful in determining if a specific host is bottlenecked. Assuming unique hosts are used for each transport, this can be helpful in determining potential transport bottlenecks. This analysis checks for average queue lengths greater than 1. <br><br>The Host Queue Length is a weighted Queue length by aggregating the record count of all the Queues (Work Q, State Q, Suspended Q) of the target host. <br><br> Reference: [BizTalk Server 2010: BizTalk Server Performance Testing Methodology](/biztalk/technical-guides/biztalk-server-performance-testing-methodology)                                                                                                                                                                                                                        |
```